src/lesson5/tv/Tv.java

```diff
@@ -0,0 +1,65 @@
+package lesson5.tv;
+
+import java.util.Objects;
+
+public class Tv {
+    private int currentChannel = 1;
+    private int currentSound = 0;
+    private final String manufacturer;
```

Comment:
Make final field first in a list, i.e. move to line 6

Reply:
ok